### DISCUSSION TITLE
2007/01/30/river-lethe

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#144] [Announcements]
# 2007/01/30/river-lethe

在一片文档里看到了这个词“River Lethe”。于是我就去网上查了一下，它的意思：

https://blog.qizhen.xyz/2007/01/30/river-lethe

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Lamby
2007年02月14日 22:32 编辑
随机路过，不知道怎么就跑这了
情人节快乐哦！

回复
Mavis
2007年02月6日 17:30 编辑
我不要喝这个水~~

回复
洋
2007年01月30日 18:01 编辑
我相信传说！！““““`

回复
奕瑋
2007年01月30日 10:46 编辑
如此雷同的叙述可能说明一个问题：
可能咱们都是真的喝了孟婆汤之后才投胎的，
所以很难想起以前的缘起和今生的目地；
或许濒死的时候又什么都记起来了，
可到那时再怎样做都无济于事了。
很多神话和传说中的真实存在，
是不以我们的相信与否而变化的。
就象你到唐朝宫廷搞一场水幕电影，
当时他们会觉得挺神奇，
再后来的人或没亲眼所见的人就会认为是幻化或天方夜谭，
再再后来的人就只能理解为神化传说了。
由于人有人的局限性，
科学也就不可避免的产生了研究的盲区和认识的误区。

回复
